apps/dynatrace/perftest/01/dynatrace-operator2.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dynatrace-operator
+  namespace: dynatrace
+spec:
+  values:
+    classicFullStack:
+      args:
+        - --set-host-group=PERF_CFT
+        - --set-network-zone=azure.cft
```

Comment:
No need for` --set-network-zone` on base its already added

Reply:
Fixed.